app/components/concept_details_component.rb

```diff
@@ -77,6 +77,31 @@ def filter_properties(top_keys, bottom_keys, exclude_keys, concept_properties)
 
   private
 
+  def modify_preflabel_key(concept_properties)
+    modified_properties = {}
+  
+    concept_properties.each do |key, value|
+      modified_key = key                                      
+      if value.is_a?(Hash) && value.key?(:key)
+        key_string = value[:key].to_s
+        modified_key = modify_key_based_on_namespace(key_string, key)
+        modified_properties[modified_key] = value unless modified_key.nil?
+      end
+    end
+  
+    return modified_properties
+  end
+  
+  def modify_key_based_on_namespace(key_string, key)
```

Comment:
you should call it `prefix_prorperty_url` and put it in the application helper, so that it can be re-used in other places

Reply:
done